daml/Marketplace/Issuance/AssetDescription.daml

```diff
@@ -25,7 +25,7 @@ template AssetDescription
     -- ^ The counterparty for any @claims@
     claims: Claims
     -- ^ Rights and obligations of the holder (and issuer)
-    safekeepingAccountId: Id
+    safekeepingAccount: Account
```

Comment:
If changing the API, please make sure you also run tests `daml test` and the UI, to check nothing breaks. The same goes for the other PR. (currently CI doesn't do this for you)

Reply:
Yes I regularly do that :)
This change was needed to be able to construct the Settlement Instructions.